terraform/alerter/lambda.tf

```diff
@@ -0,0 +1,88 @@
+# Add required providers
+terraform {
+  required_providers {
+    archive = {
+      source = "hashicorp/archive"
+      version = "2.4.2"
+    }
+  }
+}
+
+# Allow role to be assumed so lambdas can run
+data "aws_iam_policy_document" "assume_role" {
+  statement {
+    effect  = "Allow"
+    actions = [
+      "sts:AssumeRole",
+      ]
+
+    principals {
+      identifiers = ["lambda.amazonaws.com"]
+      type        = "Service"
+    }
+  }
+}
+
+# This attaches the policy needed for logging to the lambda's IAM role. #3
```

Comment:
What is the #3 on this line?

Reply:
Thas a marker for me that i forgot to remove sorry it is gone now.